flang/lib/Semantics/resolve-names.cpp

```diff
@@ -1204,7 +1204,7 @@ class DeclarationVisitor : public ArraySpecVisitor,
       const parser::Name &, const parser::Name *);
   Symbol *MakeTypeSymbol(const SourceName &, Details &&);
   Symbol *MakeTypeSymbol(const parser::Name &, Details &&);
-  bool OkToAddComponent(const parser::Name &, const Symbol * = nullptr);
+  bool OkToAddComponent(const parser::Name &, const Symbol *extends = nullptr);
```

Comment:
Is this update intentional? I don't see it being used.

Reply:
Yes, I added an argument name for documentation purposes.